yb-voyager/src/query/queryissue/issues_ddl.go

```diff
@@ -2057,3 +2057,66 @@ func NewRedundantIndexIssue(objectType string, objectName string, sqlStatement s
 	}
 	return newQueryIssue(issue, objectType, objectName, sqlStatement, map[string]interface{}{})
 }
+
+var lowCardinalityIndexIssue = issue.Issue{
+	Name:     LOW_CARDINALITY_INDEX_ISSUE_NAME,
+	Type:     LOW_CARDINALITY_INDEXES,
+	Impact:   constants.IMPACT_LEVEL_1,
+	DocsLink: "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#low-cardinality-indexes",
+}
+
+func NewLowCardinalityIndexesIssue(objectType string, objectName string, sqlStatement string, isSingleColumnIndex bool, cardinality int64, columnName string) QueryIssue {
+	issue := lowCardinalityIndexIssue
+	details := make(map[string]interface{})
+	if isSingleColumnIndex {
+		details[CALLHOME_ISSUE_NAME_KEY] = fmt.Sprintf("Low cardinality single column index with cardinality %d", cardinality)
```

Comment:
Having a "callhome" key sounds wrong here. `queryissue` layer should ideally not know/bother about callhome. The keys/values we add to the details map should be very specific to the issue at hand (no. of columns in index, for example).